asterius/src/Asterius/Builtins.hs

```diff
@@ -163,6 +163,8 @@ rtsAsteriusModule opts =
     <> tryWakeupThreadFunction opts
     <> raiseExceptionHelperFunction opts
     <> barfFunction opts
+    <> barfPushFunction opts
```

Comment:
`Asterius.Builtins` has gotten quite big, so it would be nice if we can add a new module to the `Asterius.Builtins.*` namespace, and refer to that module here.

Reply:
Agreed. I'd name the new module `Asterius.Builtins.Barf` and I'd put in it `barf`, `barf_push`, and `barf_throw` (and the respective imports). Sounds good?

Reply:
Just pushed a draft.